Logger/BaseLoggerMixins.cs

```diff
@@ -1,7 +1,38 @@
-﻿namespace Logger
+﻿using System;
+
+namespace Logger
 {
     public static class BaseLoggerMixins
     {
-        
+        public static void Error(this BaseLogger logger, string message, params Object[] arguments)
+        {
+            if (logger is null) throw new ArgumentNullException(nameof(logger));
+
+            logger.Log(LogLevel.Error, string.Format(message, arguments));
+        }
+
+        public static void Warning(this BaseLogger logger, string message, params Object[] arguments)
+        {
+            if (logger is null) throw new ArgumentNullException(nameof(logger));
+
+            logger.Log(LogLevel.Warning, string.Format(message, arguments));
+        }
+
+        public static void Information(this BaseLogger logger, string message, params Object[] arguments)

```

Comment:
```suggestion
        public static void Information(this BaseLogger? logger, string message, params object[] arguments)

```
You don't have nullability enabled in your `Logger.Tests`, but if you did you would be getting a warning when trying to pass null to these methods. To fix this warning, you would want to mark the `BaseLogger logger` parameter as nullable. This is true for all of the methods in `BaseLoggerMixins.cs`.